code/graphics/2d.cpp

```diff
@@ -995,10 +995,8 @@ bool gr_init(std::unique_ptr<os::GraphicsOperations>&& graphicsOps, int d_mode,
 
 	if (gr_get_resolution_class(width, height) != GR_640) {
 		// check for hi-res interface files so that we can verify our width/height is correct
-		bool has_sparky_hi = (cf_exists_full("2_ChoosePilot-m.pcx", CF_TYPE_ANY) && cf_exists_full("2_TechShipData-m.pcx", CF_TYPE_ANY));
-
 		// if we don't have it then fall back to 640x480 mode instead
-		if ( !has_sparky_hi ) {
+		if ( !cf_exists_full("2_ChoosePilot-m.pcx", CF_TYPE_ANY)) {
```

Comment:
Of course they weren't distributed in separate VPs; if they were, then whichever one wasn't in sparky_hi_fs2.vp should've been removed from the check ages ago. And of course the check can pass even if there are files missing for the high resolution interface; that's true of either version. All you have to do is just stick two files called 2_ChoosePilot-m.pcx and 2_TechShipData-m.pcx into an installation that's missing sparky_hi_fs2.vp and poof, FSO will try to use nonexistant high-res interface art.

Fixing *that* problem is definitely outside the scope of this PR, although I would not mind having each individual screen use either the 640x480 or 1024x768 version of its own interface art depending on whichever was available, rather than it being determined globally.